_episodes/01-intro.md

```diff
@@ -39,46 +39,92 @@ The first few rows of our first file,
 We want to:
 
 * load that data into memory,
-* calculate the average inflammation per day across all patients, and
-* plot the result.
+* calculate different statistics for all patients, and
+* plot the results.
 
 To do all that, we'll have to learn a little bit about programming.
 
 We have a dozen datasets that need analysis, stored as `.csv` files -
 but MATLAB doesn't know about these files yet.
 The first thing we need to do is set MATLAB's
 [path](reference.html#matlab-path)
-to include the directory containing the files. The MATLAB path is a list of directories
-on your computer that MATLAB knows about.
-To do this,
+to include, if necessary,  the directory containing the files. The MATLAB path is a list of directories
+on your computer that MATLAB knows about. The MATLAB path includes, by default, your current working directory (which can be displayed using command `pwd`).
+To add other directories to the MATLAB path,
 we go to the `Home` tab,
 click on `Set Path`,
 and then on `Add with Subfolders...`.
-We navigate to the directory containing our files and
+We navigate to the desired directory and
 add it to the path to tell MATLAB where to look for our files. When you refer
 to a file (either code or data), MATLAB will search all the directories in the path
 to find it. Alternatively, for data files, we can also provide the relative or
 absolute file path.
 
 > ## GNU Octave
 >
-> Octave has only recently gained a MATLAB-like user interface. To change the
-> path in any version of Octave, including command-line-only installations, use
-> `addpath('path/to/directory')`
+> Octave has only recently gained a MATLAB-like graphical user interface, and MATLAB can also be launched without it (typing `matlab -nodisplay` in a terminal). To change the
+> path in any version of Octave or MATLAB, including command-line-only sessions, use
+> `addpath('path/to/directory')`.
+>
+> There are many small differences between Octave and MATLAB, but we cannot explore all of them in this lesson. One
+> difference is that
+> the path is *not* used by Octave to search for the file given in the
+> the `csvread` command parameter. If you are using Octave you will need
+> to specify an absolute or relative path to inflammation-01.csv.
```

Comment:
Definitely -1 to bringing up `csvread` before we actually discuss it in context.